### DISCUSSION TITLE
is there a video tutorial on how to use?

### DISCUSSION THREAD

**Discussion by REDDDDY** [Mygod/VPNHotspot#383] [Q&A]
Would like to know if there is a video tutorial on how this app works and how to use it.
Searching on youtube gave random and inaccurate results.

**Comment by GaryThurman** [comment]
There's a thread on xda-developers.com that should be helpful
https://forum.xda-developers.com/t/app-root-5-0-vpn-hotspot-vpn-tethering-wi-fi-repeater-open-source.3741604/
